Tools/wasm/README.md

```diff
@@ -27,154 +27,57 @@ It comes with a reduced and preloaded stdlib without tests and threading
 support. The ``Emscripten/node`` target has threading enabled and can
 access the file system directly.
 
-Cross compiling to the wasm32-emscripten platform needs the
-[Emscripten](https://emscripten.org/) SDK and a build Python interpreter.
-Emscripten 3.1.19 or newer are recommended. All commands below are relative
-to a repository checkout.
+To cross compile to the ``wasm32-emscripten`` platform you need
+[the Emscripten compiler toolchain](https://emscripten.org/), 
+a Python interpreter, and an installation of Node version 18 or newer. Emscripten
+version 3.1.42 or newer is recommended. All commands below are relative to a checkout
+of the Python repository.
 
-#### Toolchain
+#### Install [the Emscripten compiler toolchain](https://emscripten.org/docs/getting_started/downloads.html)
 
-##### Container image
-
-Christian Heimes maintains a container image with Emscripten SDK, Python
-build dependencies, WASI-SDK, wasmtime, and several additional tools.
-
-From within your local CPython repo clone, run one of the following commands:
-
-```
-# Fedora, RHEL, CentOS
-podman run --rm -ti -v $(pwd):/python-wasm/cpython:Z -w /python-wasm/cpython quay.io/tiran/cpythonbuild:emsdk3
-
-# other
-docker run --rm -ti -v $(pwd):/python-wasm/cpython -w /python-wasm/cpython quay.io/tiran/cpythonbuild:emsdk3
+You can install the Emscripten toolchain as follows:
+```shell
+git clone https://github.com/emscripten-core/emsdk.git --depth 1
+./emsdk/emsdk install 3.1.68
+./emsdk/emsdk activate 3.1.68
```

Comment:
Any reason to nominate 3.1.68 specifically, rather than `latest`? I'm a little wary of putting a "magic number" into documentation unless that number is a minimum (like 3.1.42 above) or a pinned version of some significance.

Reply:
Okay, I'll put `latest` for now and when we add actual CI I'll switch it to the specific version used in CI.

Reply:
Out of interest - how stable/backwards compatible is "latest"? Is the emscripten version analogous to the manylinux tag/macOS dev version for compatibility purposes? Or is this a situation where we'll pick a specific emscripten version for all versions of a Python release (i.e., all 3.14.x releases are Emscripten 3.1.68)?

Reply:
If we care about abi compatibility the safe way to go is to fix a version.